kayenta-influxdb/src/main/java/com/netflix/kayenta/influxdb/canary/InfluxdbCanaryScopeFactory.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Joseph Motha
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.kayenta.influxdb.canary;
+
+import org.springframework.stereotype.Component;
+
+import com.netflix.kayenta.canary.CanaryScope;
+import com.netflix.kayenta.canary.CanaryScopeFactory;
+
+@Component
+public class InfluxdbCanaryScopeFactory implements CanaryScopeFactory {
+  @Override
+  public boolean handles(String serviceType) {
+    return serviceType.equals("influxdb");
```

Comment:
Could make the define a public static field on `InfluxdbCanaryMetricSetQueryConfig` and directly reference that constant here.